docs/archive/meeting-notes/ratify-weekly-notes-2022-Jan-2022-Jun.md

```diff
@@ -173,10 +173,10 @@ recording: https://youtu.be/mT1OVUJDg5g
 - ACTION: Jimmy will send PR to update ratify chart's service account name
```

Comment:
@susanshi  do you think if we should also update the meeting-notes?

Reply:
rollback changes in the meeting notes